settings.py

```diff
@@ -101,10 +104,12 @@ class Config:
     OMIM_TO_EFO_MAP_URL = 'https://raw.githubusercontent.com/opentargets/platform_semantic/master/resources/xref_mappings/omim_to_efo.txt'
     ZOOMA_TO_EFO_MAP_URL = 'https://raw.githubusercontent.com/opentargets/platform_semantic/master/resources/zooma/cttv_indications_3.txt'
 
-    # mouse models
-    MOUSEMODELS_PHENODIGM_SOLR = 'http://localhost:8983' # 'solrclouddev.sanger.ac.uk'
-    # TODO remove refs to user directories
-    MOUSEMODELS_CACHE_DIRECTORY = '.phenodigmcache'
+    # mouse models (Phenodigm)
+    # used to be 'http://localhost:8983' # 'solrclouddev.sanger.ac.uk'
+    MOUSEMODELS_PHENODIGM_SOLR = 'http://www.ebi.ac.uk/mi/impc'
+    # write to the cloud direcly
+    MOUSEMODELS_CACHE_DIRECTORY = 'PhenoDigm/phenodigmcache'
+    MOUSEMODELS_EVIDENCE_FILENAME = 'phenodigm-4-4-2019.json'
```

Comment:
As suggested in the previous comment, I would change this variable to:
`MOUSEMODELS_EVIDENCE_FILENAME = f"phenodigm-{datetime.today().strftime('%Y-%m-%d')}.json"`

Reply:
I would prefer to remove it from the settings file altogether and convert it into a CLI parameter as I have done in the ClinGen, Gene2Phenotype and CRISPR parsers but I accept that at least this solution is better than having it hardcoded. The only problem happens if the parser is run twice on the same day and both outputs should be kept, as the second run would replace the output of the first one. However, I don't expect this to happen very often. Something to keep in mind when for future changes.